xlsynth-g8r/src/equiv/solver_interface.rs

```diff
@@ -2035,6 +2035,34 @@ macro_rules! test_solver {
             crate::test_solver_encode!(test_xls_encode_single_hot, $solver, 8, 0x20, 3, 0x05);
             crate::test_solver_encode!(test_xls_encode_multiple_hot, $solver, 16, 0x0028, 4, 0x07);
             crate::test_solver_encode!(test_xls_encode_zero, $solver, 16, 0x0000, 4, 0x00);
+
+            use crate::xls_ir::ir;
+            use xlsynth::IrValue;
+
+            // ------------------------------------------------------------------
+            // Regression
+            // ------------------------------------------------------------------
```

Comment:
Nit: I wouldn't put big banners like // ---------

Probably just:

/// Tests that we can handle single-bit values converting to/from the solver as this has a different syntax with Boolector vs other solvers.